mods/ra/rules/misc.yaml

```diff
@@ -352,7 +349,7 @@ powerproxy.sonarpulse:
 		Icon: sonar
 		Description: Sonar Pulse
 		LongDesc: Reveals all submarines in the vicinity for a \nshort time.
-		ChargeTime: 30
+		ChargeTime: 20
```

Comment:
Now that I buffed it to not reveal without other (naval) units I thought that would be okay. Removed the change.